Jenkinsfile

```diff
@@ -9,6 +9,9 @@ pipeline {
   libraries {
     lib('fxtest@1.10')
   }
+  triggers {
+    pollSCM('*/5 * * * *')
```

Comment:
Can we use 'H/5 * * * *' so not all jobs will be trying to poll at the same time? I wonder if every 5 minutes is too frequent, depending on how many jobs we have.